pkg/azuredisk/identityserver.go

```diff
@@ -49,7 +50,7 @@ func (f *Driver) GetPluginInfo(ctx context.Context, req *csi.GetPluginInfoReques
 // Currently the spec does not dictate what you should return either.
 // Hence, return an empty response
 func (f *Driver) Probe(ctx context.Context, req *csi.ProbeRequest) (*csi.ProbeResponse, error) {
-	return &csi.ProbeResponse{}, nil
+	return &csi.ProbeResponse{Ready: &wrappers.BoolValue{Value: true}}, nil
```

Comment:
any suggestion on what are the checks we should perform to conclude the CSI driver to be healthy?

Reply:
cc @andyzhangx

Reply:
@ashishranjan738 I think we can leave it now as I don't have idea how to implement it now. let's keep this issue open. Thanks.